models/device.py

```diff
@@ -121,39 +122,43 @@ def note(self, uuid, note):
             }
             return self.base_request.request('device', 'PATCH', params=params, data=data, endpoint=self.settings.get('pine_endpoint'))
         else:
-            print(Message.NO_DEVICE_FOUND.format(value=uuid, dev_att="uuid"))
+            raise ResinException.ResinDeviceNotFound(uuid)
 
     def get_display_name(self, device_type_slug):
-        device_type_found = self.config.get_device_type()
+        device_type_found = self.config.get_device_types()
```

Comment:
here and above the var name looks invalid, it reads as boolean, and has the "type" in singular. It's also only used once, so the method call can be in comprehension. BTW I think the `get_device_types` method chouls cache its results, and basically the Config class itself should only call the /config endpoint once because it's changing very seldom, so usually no need to re-fetch it throughout the app liefcycle